arelle/Cntlr.py

```diff
@@ -234,7 +234,7 @@ def __init__(self, hasGui=False, logFileName=None, logFileMode=None, logFileEnco
             }
 
         # start language translation for domain
-        self.setUiLanguage(self.config.get("userInterfaceLangOverride",None), fallbackToDefault=True)
+        self.setUiLanguage(uiLang or self.config.get("userInterfaceLangOverride",None), fallbackToDefault=True)
```

Comment:
Could this be moved a few lines below after `self.modelManager` is assigned to avoid the `hasattr(self, "modelManager")` check in `self.setUiLanguage`?